### ISSUE TITLE
Support for DJI Mini 4 Pro

### ISSUE THREAD

**Issue by KennethWilke** [dji-sdk/Mobile-SDK-Android-V5#400]
I love DJI products and want to buy the Mini 4 Pro to use with the SDK, but it seems like the best sub-$12,000, in-stock drone with an SDK is the Mini 3. Since the Mini 3 Pro got an SDK after being out like 11 months, maybe based on a similar cadence we'd get an SDK for the Mini 4 Pro by roughly the end of this year?

For any other folks in a similar situation, have you had any luck switching to Autel Robotics or any other brands?

**Comment by RyannGalea** [comment]
Yes it seems very odd that we still don't have the support yet. Very frustrating.

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/114412'>Zendesk ticket #114412</a>:<br><br>We are considering adapting the DJI MINI 4 PRO. However, a complete and clear adaptation plan has not yet been ready, so we are currently unable to gain an exact timeline.
<br><p><em><small>°°°</small></em></p>

**Comment by RyannGalea** [comment]
Wow, so not even close. Disappointing.

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/114412'>Zendesk ticket #114412</a>:<br><br>We apologize for any inconvenience caused.
<br><p><em><small>°°°</small></em></p>

**Comment by jwcharp** [comment]
Can we please get support for this @dji-dev

**Comment by mulkatz** [comment]
👍 for mini 4 pro support!

**Comment by picofpv** [comment]
The DJI Mini 4 Pro uses the RCN2 and RC2 model remote controllers.I guess it will take a long time to adapt the SDK to these new remote controls

**Comment by P-Mindz** [comment]
@dji-dev 
What do you mean by considering? You already announced that a SDK will be available within the first half of 2024. Now 2024 ist ending and we still have no SDK or even a timeline!

**Comment by joaquinperaza** [comment]
Can we have an answer on this? There even was an active issue that was closed without an answer #164

**Comment by chriswingler** [comment]
Bump

**Comment by ian-mcpherson** [comment]
Like many others I have been waiting for the SDK for the Mini 4 Pro. There appears to have been a change of heart from DJI about releasing this, I am wondering if this has anything to do with what the US is doing to DJI. The DJI ban.

**Comment by KennethWilke** [comment]
Looks like Anzu is starting to work to fill the gap here, DroneDeploy just announced support for their drones. I haven't seen the SDK stuff yet, but at least some folks are starting to fill the demand

**Comment by chriswingler** [comment]
Bump. Please release SDK that was promised first half of this year.

**Comment by joaoramosipl** [comment]
Just bought 2 drones for development. Really need SDK to incorporate these in our Mavic 2 Enterprise Advanced + DJI P4 fleet for autonomous area mapping.

**Comment by jwcharp** [comment]
We are currently facing supply challenges that are impacting our customers. At the moment, we recommend the DJI Mini 3 Pro, but unfortunately, it is becoming increasingly difficult to find or purchase.

We require a drone with the higher 100-120FPS capabilities that the Mini 3 Pro offers. However, the DJI Mini 4 Pro, is also becoming scarce and does not have mSDK support, which is needed for our Android App to function.

We noticed the release of the DJI Flip. Could you confirm whether mSDK support will be extended to this device? This would greatly help us continue to develop and expand our customer base and products.

**Comment by StrandedTurtle** [comment]
Hi is there any news on this. I've found countless examples of you saying it will be released in 2024. Just give us confirmation that it is still coming. An £1000 drone should absolutely have sdk support. Furthermore this should also include the £300 PRO controller. 

@dji-dev 2 weeks later and no response. Give the people an update and be transparent. If you're not going to make one at least be honest with your customers and tell us.

**Comment by KennethWilke** [comment]
Looks like we got the support added with the Mobile SDK version 5.13! Awesome to see

**Comment by jwcharp** [comment]
Yes I saw this, unfortunately it's not plug and play with the mini 3 pro, or somethings are broken. I am having issues with getting the feed to display and the gimbal to move, if anyone has success on these please let me know what you code looks like to display the feed with the mini 4 pro

**Comment by chriswingler** [comment]
great job folks keepin the dream alive

**Comment by billyandriam** [comment]
@jwcharp Did you manage to make it work now one month later? I am planning to upgrade to Mini 4 Pro so your input is highly valued.

**Comment by jwcharp** [comment]
@billyandriam Yes we got it working to show the feed, however we are having issues now with the Virtual Stick commands.
When issuing a constant velocity say of 1 m/s the 4 Pro studders along the way, while the 3 Pro is completely smooth - we adjusted the command frequency to max and min recommended rates and still no resolve. We have sent DJI our drone logs they are investigating - curious if anyone else has V-Stick issue with the 4 Pro
